src/components/AssessmentDetailsPanel.tsx

```diff
@@ -119,9 +126,48 @@ const ArNSAssessmentPanel = ({
             </button>
           </div>
           {timingsPanelOpen && arnsAssessment.timings && (
-            <div className="flex gap-1 px-3 pb-3">
-              <TimerIcon className="size-4" />
-              <div>Total: {arnsAssessment.timings.total} ms</div>
+            <div className="flex flex-col gap-1 px-3 pb-3">
+              <div className="flex gap-1">
+                <TimerIcon className="size-4" />
+                <div className="text-high">
+                  Total: {arnsAssessment.timings.total} ms
+                </div>
+              </div>
+              {arnsAssessment.timings.dns !== undefined && (
```

Comment:
Seems like theres some opportunity to DRY this a bit. Repeated use of the object key to display the object value in repeated div tags of the same class is good use case for an entries().map

Reply:
updated code